src/results/proforma.jl

```diff
@@ -297,6 +370,12 @@ function update_metrics(m::Metrics, p::REoptInputs, tech::AbstractTech, tech_nam
     m.om_series += escalate_om(annual_om)
     m.om_series_bau += escalate_om(-1 * existing_kw * tech.om_cost_per_kw)
 
+    if tech_name == "CHP"
+        escalate_fuel(val, esc_rate) = [val * (1 + esc_rate)^yr for yr in 1:years]
```

Comment:
so there wasn't a way to make these functions globally available so they aren't defined twice?

Reply:
Right, I guess these are "local scope" functions when they are defined within the function. I don't think they will ever change though, so low risk of these becoming obsolete when the other function is updated.